patches/chrome-browser-profiles-profile_destroyer.cc.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/chrome/browser/profiles/profile_destroyer.cc b/chrome/browser/profiles/profile_destroyer.cc
-index 517fd2a8a924f3076ad14105a0ec2677e18b0c79..1abc39094e3ffa59725dbeec543eac4371b81d18 100644
+index 80047fddb2877428d072e55f5ec5631590328c30..b73e30ad3eb3e81c34299308a7072be92166fcbd 100644
 --- a/chrome/browser/profiles/profile_destroyer.cc
 +++ b/chrome/browser/profiles/profile_destroyer.cc
-@@ -88,6 +88,7 @@ void ProfileDestroyer::DestroyProfileWhenAppropriate(Profile* const profile) {
+@@ -104,6 +104,7 @@ void ProfileDestroyer::DestroyProfileWhenAppropriateWithTimeout(
    // ignored during shutdown and by the System Profile do not either.
    HostSet profile_hosts;
    GetHostsForProfile(&profile_hosts, profile);
 +  BRAVE_PROFILE_DESTROYER_DESTROY_PROFILE_WHEN_APPROPRIATE
```

Comment:
This fix is likely not needed any more. This was fixed upstream via https://bugs.chromium.org/p/chromium/issues/detail?id=1357476 and merged to M106.

Reply:
Removed here 484120f12f0306c2657794dd03e01d6c22d27140.